config/cesm/config_grids.xml

```diff
@@ -1233,41 +1233,40 @@
+
       <file grid="atm|lnd" mask="tnx1v1">domain.lnd.fv0.9x1.25_tnx1v1.150512.nc</file>
       <file grid="ocnice"  mask="tnx1v1">domain.ocn.0.9x1.25_tnx1v1.150512.nc</file>
-      <file grid="atm|lnd" mask="tnx1v3">domain.lnd.fv0.9x1.25_tnx1v3.170621.nc</file>
-      <file grid="ocnice"  mask="tnx1v3">domain.ocn.tnx1v3.170621.nc</file>
+
       <file grid="atm|lnd" mask="tnx1v4">domain.lnd.fv0.9x1.25_tnx1v4.170609.nc</file>
-      <file grid="ocnice"  mask="tnx1v4">domain.ocn.tnx1v4.170606.nc</file>
+      <file grid="ocnice"  mask="tnx1v4">domain.ocn.fv0.9x1.25_tnx1v4.170609_djlo.nc</file>
+
       <file grid="atm|lnd" mask="tnx0.25v1">domain.lnd.fv0.9x1.25_tnx0.25v1.140618.nc</file>
-      <file grid="atm|lnd" mask="tnx0.25v3">domain.lnd.fv0.9x1.25_tnx0.25v3.170629.nc</file>
-      <file grid="ocnice"  mask="tnx0.25v3">domain.ocn.tnx0.25v3.170629.nc</file>
-      <file grid="atm|lnd" mask="tnx0.25v4">domain.lnd.fv0.9x1.25_tnx0.25v4.170629.nc</file>
-      <file grid="ocnice"  mask="tnx0.25v4">domain.ocn.tnx0.25v4.170629.nc</file>
-      <file grid="atm|lnd" mask="tnx0.125v4">domain.lnd.fv0.9x1.25_tnx0.125v4.200830.nc</file>
-      <file grid="ocnice"  mask="tnx0.125v4">domain.ocn.tnx0.125v4.200725.nc</file>
-      <!--file grid="ocnice" mask="tnx0.25v1">domain.ocn.0.9x1.25_tnx0.25v1.140618.nc</file-->
+      <file grid="ocnice"  mask="tnx0.25v1">domain.ocn.0.9x1.25_tnx0.25v1.140618.nc</file>
+
+      <!-- <file grid="atm|lnd" mask="tnx0.25v3">domain.lnd.fv0.9x1.25_tnx0.25v3.170629.nc</file> -->
+      <!-- <file grid="ocnice"  mask="tnx0.25v3"></file> -->
+
+      <!-- <file grid="atm|lnd" mask="tnx0.25v4">domain.lnd.fv0.9x1.25_tnx0.25v4.170629.nc</file> -->
+      <!-- <file grid="ocnice"  mask="tnx0.25v4"></file> -->
+
+      <!-- <file grid="atm|lnd" mask="tnx0.125v4">domain.lnd.fv0.9x1.25_tnx0.125v4.200830.nc</file> -->
+      <!-- <file grid="ocnice"  mask="tnx0.1.25v4"></file> -->
       <desc>0.9x1.25 is FV 1-deg grid:</desc>
     </domain>
 
     <domain name="1.9x2.5">
       <nx>144</nx>  <ny>96</ny>
       <file grid="atm|lnd" mask="gx1v6">domain.lnd.fv1.9x2.5_gx1v6.090206.nc</file>
       <file grid="ocnice"  mask="gx1v6">domain.ocn.1.9x2.5_gx1v6_090403.nc</file>
+
       <file grid="atm|lnd" mask="gx1v7">domain.lnd.fv1.9x2.5_gx1v7.170518.nc</file>
       <file grid="ocnice"  mask="gx1v7">domain.ocn.fv1.9x2.5_gx1v7.170518.nc</file>
-      <file grid="ocnice"  mask="null">domain.aqua.fv1.9x2.5.nc</file>
-      <file grid="atm|lnd" mask="tnx1v1">domain.lnd.fv1.9x2.5_tnx1v1.120120.nc</file>
-      <file grid="atm|lnd" mask="tnx1v3">domain.lnd.fv1.9x2.5_tnx1v3.170622.nc</file>
-      <file grid="ocnice"  mask="tnx1v3">domain.ocn.tnx1v3.170622.nc</file>
+
       <file grid="atm|lnd" mask="tnx1v4">domain.lnd.fv1.9x2.5_tnx1v4.170609.nc</file>
-      <file grid="ocnice"  mask="tnx1v4">domain.ocn.tnx1v4.170606.nc</file>
+      <file grid="ocnice"  mask="tnx1v4">domain.ocn.fv1.9x2.5_tnx1v4.170609_djlo.nc</file>
```

Comment:
Do we need Dirk's initials on this file?

Reply:
I'd prefer not to have that - but I'm not comfortable changing the file names right now.

Reply:
Okay